includes/widgets/video.php

```diff
@@ -277,6 +277,10 @@ protected function register_controls() {
 				'label' => __( 'Autoplay', 'elementor' ),
 				'type' => Controls_Manager::SWITCHER,
 				'frontend_available' => true,
+				'condition' => [
```

Comment:
Fix the condition, and make sure all providers start to play after clicking, in case of image overlay

Reply:
Done.